test/functional/provider/python3_spec.lua

```diff
@@ -90,6 +90,11 @@ describe('python3 provider', function()
     eq({1, 2, {['key'] = 'val'}}, eval([[py3eval('[1, 2, {"key": "val"}]')]]))
   end)
 
+  it('pyxeval #10758', function()
```

Comment:
could check `eq(0, eval([[&pyxversion]]))` as a precondition (setup did not use pyx by accident, or something)